org.gnome.Raider.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "org.gnome.Raider",
```

Comment:
You should change it to `io.github.ADBeveridge.Raider` and rename desktop, icon and appdata files in your repo too. `org.gnome` is reserved for official GNOME projects.

Reply:
the code needs to be changed too in that case:

https://github.com/ADBeveridge/raider/blob/main/src/raider.c#L109

Reply:
This program will be officially GNOME when I can finish adding it to Flathub. The reason why is that when submitting a new program to the GNOME [Circle](circle.gnome.org) the project_application issue page asks for a Flathub page. That is why my program is named org.gnome.Raider, but not yet part of GNOME. The reason why I do not name it com.github.Raider instead, and when GNOME Circle accepts the program, I rename it is because it would be a hassle.

Reply:
But surely the flathub bot cannot fail just because of the name of the program? It it says that the appdata file needs a description tag, and [it](https://github.com/ADBeveridge/raider/blob/main/data/org.gnome.Raider.appdata.xml.in) does. What am I missing? Is it looking at some other file?

Reply:
I don't think GNOME Circle apps get the right to use `org.gnome`. You are still expected to use your domain (either custom or github/gitlab-based). In this case, it would be `com.github.ADBeridge.Raider`.

The build fails because the appdata file fails validation:

```
Processing application org.gnome.Raider
org.gnome.Raider:                  AppData problem: tag-missing : <description> required
```

Reply:
OK, I will try it out. Have a happy new year!!! :)

Reply:
I changed the appdata file, raider.c, and the manifest.